### DISCUSSION TITLE
Buff icon when 2xp is active

### DISCUSSION THREAD

**Discussion by jeyrella** [chromiecraft/chromiecraft#4549] [Suggestions]
Add a "buff" icon when 2xp is active.  Wouldn't be an actual buff, just an indicator.  Bonus if this is unshown if player manually deactivates 2xp through the console.
This would drastically reduce (and eventually stop)  the "is the double xp on" spam every time someone logs in (from experience).

**Comment by 55Honey** [comment]
Hello jeyrella,
thanks for your suggestion.

Custom buffs or custom icons in general tend to require custom clients, which is a thing we can't do.
But you can already see a message when you log in. `It's the weekend!` will display, when the 2x quest xp is up.

I hope that clarifies your doubts.

Regards, Honey

**Comment by Nyeriah** [comment]
Closing this as the 2x experience is active all the time (you can still choose lower rates if you wish). Therefore, this suggestion is outdated
